.devcontainer/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM centos/devtoolset-6-toolchain-centos7:latest
+
+LABEL \
+  description="Ovsrpro Centos 7 Development Container" \
+  name="ovsrpro-dev"
+
+USER 0
+RUN set -eux; \
+  yum --assumeyes update; \
+  yum install --assumeyes \
+    epel-release \
+    # This repository gives us more modern versions of some software, such as Git.
+    https://packages.endpoint.com/rhel/7/os/x86_64/endpoint-repo-1.7-1.x86_64.rpm; \
+  yum install --assumeyes \
+    cmake3 \
+    git \
+    make \
+    wget; \
+  # yum --assumeyes clean all; \
+  ln -s cmake3 /usr/bin/cmake; \
+  ln -s ctest3 /usr/bin/ctest; \
+  ln -s cpack3 /usr/bin/cpack; \
+  yum install --assumeyes \
+    # Required by GLEW
+    libX11-devel \
+    mesa-libGL-devel \
+    mesa-libGLU-devel \
+    # Required for Qt5
+    bison \
+    expat-devel \
+    fontconfig-devel \
+    freetype-devel \
+    gperf \
+    libxcb-devel \
+    libxkbcommon-devel \
+    openssl-devel \
+    which \
```

Comment:
I am surprised `which` is not installed by default on a RHEL or CentOS system.

Reply:
So was I the first time that I encountered that missing gem.